resources/default_pipeline_environment.yml

```diff
@@ -238,6 +238,8 @@ steps:
       workspace: 'nohup.out'
   dubExecute:
     dockerImage: 'dlang2/dmd-ubuntu:latest'
+  githubCreateIssue:
+    assignees: []
```

Comment:
Setting the default could also be done in the metadata yml for the step.

Reply:
yes, we should do it in the step defaults ...

Reply:
```suggestion
```

Reply:
Totally missed it on the first go. Fixed it. Thanks ☺️